src/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoader.java

```diff
@@ -410,7 +410,8 @@ public IInternalContest fromYaml(IInternalContest contest, String[] yamlLines, S
             setCcsTestMode(contest, ccsTestMode);
         }
         
-        boolean loadSamples = fetchBooleanValue(content, LOAD_SAMPLE_JUDGES_DATA, true);
+        // TODO 701 change LOAD_SAMPLE_JUDGES_DATA, false); to , true);
+        boolean loadSamples = fetchBooleanValue(content, LOAD_SAMPLE_JUDGES_DATA, false);
```

Comment:
it might be safer to change this to loadSamples = false,  we know we do not want to enable this broken implementation.

Reply:
This is a non-documented change, so only us (or those who happen to read the code) would enable it.

Reply:
This change allows the person who update the code to test it with a single .ini entry.